### PR TITLE
Get rid of x-products discrepancies

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

File: openapi/paths/shipping-rates.yaml [hunk 1]
```diff
@@ -2,8 +2,9 @@ parameters:
   - $ref: ../components/parameters/organizationId.yaml
 get:
   x-products:
-    - Full
     - Core
+    - Full
+    - Billing
```
Comment (ashkarpetin) [inline openapi/paths/shipping-rates.yaml:7]: I don't see shipping rates tags here: https://github.com/Rebilly/api-definitions/blob/f87e51d54b15fb1e295b853318c8d4e53f56d8ec/plugins/products-bundler/mapping/Billing.yaml#L2

File: openapi/paths/applications.yaml [hunk 2]
```diff
@@ -48,6 +48,9 @@ get:
       source:
         $ref: ../code_samples/JavaScript/applications/get.js
 post:
+  x-products:
+    - Users
+    - Full
```
Comment (ashkarpetin) [inline openapi/paths/applications.yaml:53]: I don't see `Application users` tags in Full

File: openapi/paths/digital-wallets@onboarding@apple-pay.yaml [hunk 3]
```diff
@@ -1,4 +1,8 @@
 post:
+  x-products:
+    - Core
```
Comment (ashkarpetin) [inline openapi/paths/digital-wallets@onboarding@apple-pay.yaml:3]: There is no  `Gateway Accounts` tag in `Core`

File: openapi/paths/shipping-rates.yaml (hunk 1 above)
Comment (VladimirBerdnik) [reply openapi/paths/shipping-rates.yaml:7]: You're right, thank you!

**Review by ashkarpetin — APPROVED** [review]

**Review by n-fasano — APPROVED** [review]

**Pull request merged**